manifests/service.pp

```diff
@@ -22,4 +22,7 @@
     }
   }
 
+  if $candlepin::amq_enable {
+    Qpid::Config::Bind <| tag == 'qpid' |> -> Service <| tag == 'tomcat' |>
```

Comment:
I think you can write the last one as `Service['tomcat']`

Reply:
Yeah, unless we're running in a container and that Service is not defined at all. This way it just collects nothing :)

Reply:
I'm still very hesitant about allowing the use of containers in this module. I need some time to process and gather my thoughts on it.